flowzone.yml

```diff
@@ -571,6 +583,18 @@ jobs:
           cmd: bash -c "echo $INPUT | tr -d '[:space:]'"
           separator: ","
 
+      # Check if we have atleast a README to build a website from
+      - name: Check for README for building a website
+        id: cloudflare_website
+        run: |
+          if test -f "README.md"
```

Comment:
You're going to build a cloudflare page for every single repo we have? I believe they all have a readme (I think balenaCI required a `README.md` even if it was empty). Maybe that makes sense but I'm not sure

Reply:
@Page- it requires at least a readme and most definitely an input called "cloudflare_website" in your repo level flowzone.yml
I am checking for only the readme in this part of the yml - and checking for the input further down.

Most definitely don't intend to build a site for each repo. Only ones that want that and express so through their inputs

Reply:
PS - do you suggest I check for the input at the top instead of at the bottom right before I do the actual building?

Reply:
I think renaming this to `has_readme` might make sense and then the `website_build` is `has_readme && cloudflare_website != ''`

Reply:
Sounds good!